src/commands/service/list.ts

```diff
@@ -15,28 +16,12 @@ export default class List extends Command {
     }
 
     const { username, auth } = config
+    let api = new JamsocketApi(auth);
 
-    const endpoint = `${API}${getServiceListEndpoint(username)}`
-    const response = await request(endpoint, null, {
-      headers: { 'Authorization': `Basic ${auth}` },
-    })
-
-    let responseBody
-    try {
-      responseBody = JSON.parse(response.body)
-    } catch (error) {
-      this.error(`jamsocket: error parsing JSON response - ${error}: ${response.body}`)
-    }
-
-    if (response.statusCode && response.statusCode >= 400) {
-      const { message, status, code, id } = responseBody.error
-      this.error(`jamsocket: ${status} - ${code}: ${message} (id: ${id})`)
-    } else {
-      // TODO: print a table that gives information about each service
-      const services = responseBody.services
-      for (const service of services) {
-        this.log(service)
-      }
+    let responseBody = await api.serviceList(username);
```

Comment:
Nit: make these `const`s instead of `let`s.

(We should probably just set up prettier at some point to do these changes for us automatically.)